src/lib/components/InstallPlugin.svelte

```diff
@@ -9,10 +9,22 @@
   }
 
   const create = $derived(`conda env create --name ${env_name} --file ${env_url}`);
+  const create_silicon = $derived(
+    [
+      "CONDA_SUBDIR=osx-64 conda env create",
+      `--name ${env_name}`,
+      `--file ${env_url}`,
+      `conda activate ${env_name}`,
+      "conda config --env --set subdir osx-64"
+    ].join('\n')
+  );
   const update = $derived(`conda env update --file ${env_url}`);
 
   const ast = $derived(u('div', key({class: '-my-5'}), [u('tabSet', key(), [
-      u('tabItem', key({title: '[Fresh Install]'}), [
+      u('tabItem', key({title: '[Fresh Install (Apple Silicon)]'}), [
+          u('code', key({lang: 'bash', class: '!my-3'}), create_silicon),
+      ]),
+      u('tabItem', key({title: '[Fresh Install (all other architectures)]'}), [
```

Comment:
The titles end up being pretty long, I wonder if we can just have an exception for `(Apple Silicon)`?

Reply:
yeah, i know... i do worry about it being confusing if we only have one arch specified. let me think about other name options to see if there's still a way to convey this that's less verbose

Reply:
<img width="570" height="235" alt="Screen Shot 2025-07-10 at 4 23 04 PM" src="https://github.com/user-attachments/assets/83d04794-a3be-44ee-ba5c-a1688f52bee5" />

How do you feel about this @ebolyen?